programmers/Wooahan-agile-homework4/로또의 최고 순위와 최저 순위.js

```diff
@@ -0,0 +1,21 @@
+// 로또의 최고 순위와 최저 순위
+
+function solution(lottos, win_nums) {
+    let answer = [];
```

Comment:
```suggestion
    const answer = [];
```
배열 자체가 바뀌는 경우가 많지 않기때문에 보통 const로 선언